client/my-sites/plans/ecommerce-trial/entrepreneur-plan/entrepreneur-plan.tsx

```diff
@@ -32,15 +51,15 @@ const useEntrepreneurPlanPrices = () => {
 		PLAN_ECOMMERCE: {
 			term: translate( 'Pay yearly' ),
 			slug: PLAN_ECOMMERCE,
-		},
+		} as PlanPriceType,
```

Comment:
Instead of using `as` for each object, could we do:
```
const planPrice: Record<string, PlanPriceType> = { ... };
```

Reply:
Or to be more precise:
```
const planPrices: Record< PlanKeys, PlanPriceType > = {...} 
```

Reply:
Sure, that is a good use of Typescript!